src/components/inline/InputText/InputText.stories.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+import InputText from "./InputText";
+
+const demoStyling = {
+  left: "50%",
+  position: "absolute",
+  top: "50%",
+  transform: "translate(-50%, -50%)",
+  width: "200px",
+};
+
+export default {
+  title: "Inline/InputText",
+  component: InputText,
+};
+
+const [text, setValue] = useState("");
```

Comment:
This is not legal here. `useState()` only works inside of function components.

Is there a reason you keep trying to define this like this instead of like a normal component?

Reply:
ok moving back